web/unit_tests/test_utils.js

```diff
@@ -30,35 +46,73 @@ var setupKMW = function(ui) {
    */
   window.setTimeout(function() {
     initFunc();
-  }, 5);
+  }, 8);
```

Comment:
Why the change from 5 to 8? Given `setTimeout` has a minimum timeout of 10, what was the purpose of this?

Reply:
I was tweaking that value when figuring out the source of some weird UI interactions when (as it turns out) the post-init timeouts were too low.  I'll put it back.